projects/uitdatabank/models/common-name-localized.json

```diff
@@ -3,6 +3,7 @@
   "title": "name.localized",
   "description": "A human-readable name localized in a single value.",
   "minLength": 1,
+  "maxLength": 90,
```

Comment:
Which schemas use this name? Can it cause other fields (accept the name of a place, event and organizer) to have an unwanted limit?

Reply:
I checked and no other usage was detected